assets/src/dashboard/app/views/savedTemplates/index.js

```diff
@@ -67,9 +72,22 @@ function Header({ search, stories, view, sort }) {
         stories={stories}
         handleTypeaheadChange={search.setKeyword}
         typeaheadValue={search.keyword}
-      />
+      >
+        <HeaderToggleButtonContainer>
+          <ToggleButtonGroup
+            buttons={SAVED_TEMPLATES_STATUSES.map((savedTemplateStatus) => {
+              return {
+                handleClick: () => filter.set(savedTemplateStatus.value),
+                key: savedTemplateStatus.value,
+                isActive: filter.value === savedTemplateStatus.value,
+                text: savedTemplateStatus.label,
+              };
+            })}
+          />
+        </HeaderToggleButtonContainer>
```

Comment:
We can drop this for now since we will not have the toggle buttons initially.

Reply:
will do!